src/Controller/Controller.php

```diff
@@ -197,13 +197,20 @@
             }
         });
 
+        if ($this->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_REMEMBERED')) {
+            $user = $this->get('security.token_storage')->getToken()->getUser();
+            $profile = $this->get('elife.api_sdk.profiles')->get($user->getUsername());
```

Comment:
`getUsername()` is using some standard Symfony terminology but it's the profile id right?

Reply:
Yep.